src/PrestaShopBundle/Form/Admin/Product/ProductPrice.php

```diff
@@ -212,6 +218,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'id_product' => 1,
```

Comment:
Is there a reason to use the product id `1`? What happened if this product doesn't exists?

Reply:
The `1` is here in order to maintain the legacy behavior of this Form. This is a default value. I could put it in a `const` maybe 🤔